src/com/hms/flexyosisoftconnector/OSIsoftServer.java

```diff
@@ -542,8 +547,8 @@ public static boolean postOcsBatch(String payload) {
    * @return returns the http response code
    */
   public static boolean postBatch(String payload) {
-    int res = NO_ERROR;
-    res =
+    boolean requestSuccess = false;
+    requestSuccess =
         RequestHTTPS(

```

Comment:
```suggestion
    boolean requestSuccess = RequestHTTPS(
```
why not inline the value? false is redundant

Reply:
Okay yeah, I when i was messing with git it went back to returning true instead of the variable which is why this looks weird. It won't be useless when I change that

Reply:
also made it inline for this and for the other two as well :)